docs/codeql/ql-language-reference/ql-language-specification.rst

```diff
@@ -502,7 +504,7 @@ An named tuple *directly satisfies* a predicate or type with a given tuple if th
 
 A value ``v`` is in a type ``t`` under any of the following conditions:
 
--  The type of ``v`` is ``t`` and ``t`` is a primitive type.
+-  The type of ``v`` is ``t`` and ``t`` is a primitive type (except :ref:`QlBuiltins::BigInt <bigint>`).
```

Comment:
Yeah, this is one of those awkward cases where the base case is hard to write because it is almost self referential but `BigInt` definitely should be included here.

Reply:
Yep, you're totally correct, I misinterpreted "store" to mean "dbscheme".